web/charts/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Charts-Flutter Demo
 publish_to: none
 
 environment:
-  sdk: '>=2.10.0 <3.0.0'
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
nit: we're still fetching `charts_flutter` from GitHub - it looks like we could import it from Pub now?

Reply:
The latest published version is broken w.r.t Flutter 2.5. Stuart Morgan is in the process of rolling up a release of `charts_flutter` that will enable me to revert this git dependency and then we will look at how to support the package going forward.